index.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Asad Shareef Protfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="container">
+    <header>
+        <a href="">My Logo</a>
+        <ul>
+            <li>Protfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+        </ul>
+        <nav class="flex-nav"><button>☰</button></nav>
+    </header>
+    <div class="headline">
+        <h1>I’m Asad Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p id="social-title">LET’S CONNECT</p>
+        <ul>
+            <li class="social" ><a href=""><img src="assets/twitter.svg" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Linkedin icon.svg" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Vector.png" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Vector2.png" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Vector3.png" alt=""></a></li>
+        </ul>
+    </div>
+    </section>
```

Comment:
The following are the issues found in the headline section:

- Kindly use the correct colors for the lets connect in order to make it match the Figma design and make it more visually appealing.

- I also suggest that the let's connect should be a clickable link because when it is clicked it should lead to the contact form which will make the page easily navigatable.

- To make the UI look nicer, consider adding the background images on the headline section and also the rounded edge at the bottom left of the section.


Notice the highlighted section  in the screenshot below


YOUR DESIGN

![no nothing](https://user-images.githubusercontent.com/84930355/212212782-628bf8db-7c33-4aa8-8ef9-fd9e9f7d1b8a.PNG)


FIGMA DESIGN

![good bg](https://user-images.githubusercontent.com/84930355/212213331-6ff9fc14-f0e3-484d-81a1-567b6d55873c.PNG)